.travis.yml

```diff
@@ -52,7 +52,8 @@ jobs:
     # variables and are therefore not available for forks of the repository.
     # For more details, refer to https://docs.travis-ci.com/user/environment-variables#encrypting-environment-variables
     if: branch = develop AND type = push
-    env: RUN_E2E_TESTS_MOBILE=true
+    env: RUN_E2E_TESTS_MOBILE=false
+    # TODO(apb7): Enable mobile tests. See Issue #5839.
```

Comment:
TODOs are usually above where the action needs to be taken right?

Reply:
Shall I put it before the if branching?

Reply:
Or better so, on the same line as the `env` thing?

Reply:
Oh the env has to follow the if? In that case I think it is fine as is....